### PR TITLE
fix example to match figma

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR aligns the center layer example with the figma file
There were 3 key things missing. 
I will need to get the updated icon this is not yet in Grommet Icons

- [X] Add "x" close button in the upper right have corner
- [X] Left aligned text

#### Where should the reviewer start?
LayerCenterExample
#### How should this be manually tested?
https://www.figma.com/file/Ts53TAipMolmsv9DxWG3p0/HPE-Layer-Component?node-id=100%3A70
The example should match the figma fle
#### What are the relevant issues?
Issue #1426
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
yes

### CODE REVIEW

File: aries-site/src/examples/components/layer/LayerCenterExample.js [hunk 1]
```diff
@@ -18,27 +18,36 @@ export const LayerCenterExample = () => {
...
-                <MailOption />
-              </Box>
-              <Box gap="medium">
-                <Heading margin="none" size="small">
+            <Box justify="between" direction="row">
+              <Box flex={false} gap="small" direction="row">
+                <Box justify="center">
+                  <MailOption />
+                </Box>
+                <Text margin="none" size="xlarge">
```
Comment (taysea) [inline aries-site/src/examples/components/layer/LayerCenterExample.js:28]: Any reason to not use Heading?

File: aries-site/src/examples/components/layer/LayerCenterExample.js [hunk 2]
```diff
@@ -18,27 +18,36 @@ export const LayerCenterExample = () => {
...
+                For modal dialogs, the use case will determine the design and
+                size of the box for your content
+              </Text>
             </Box>
+          </Box>
+          <Box
+            fill="vertical"
+            overflow="auto"
+            width={size !== 'small' ? 'medium' : undefined}
+          >
```
Comment (taysea) [inline aries-site/src/examples/components/layer/LayerCenterExample.js:50]: Can't comment on line 54, but I think the pad should be `{{ vertical: 'small', horizontal: 'medium' }}`

File: aries-site/src/examples/components/layer/LayerCenterExample.js [hunk 3]
```diff
@@ -18,27 +18,36 @@ export const LayerCenterExample = () => {
...
                   Modal Dialog
-                </Heading>
-                <Text>
-                  For modal dialogs, the use case will determine the design and
-                  size of the box for your content
                 </Text>
               </Box>
+              <Box justify="center">
+                <Button
+                  icon={<Close size="small" align="center" />}
```
Comment (taysea) [inline aries-site/src/examples/components/layer/LayerCenterExample.js:34]: The icon doesn't look aligned when you hover over it. Do you notice that? Seems like `FormClose` icon works better and aligns with what we're doing in Filtering.

Comment (britt6612) [reply aries-site/src/examples/components/layer/LayerCenterExample.js:34]: okay let me change that and see I did notice the alignment was off without the align which didn't feel correct

File: aries-site/src/examples/components/layer/LayerCenterExample.js (hunk 1 above)
Comment (britt6612) [reply aries-site/src/examples/components/layer/LayerCenterExample.js]: You're correct I changed it  back but used level 3 to match figma

**Review by taysea — APPROVED** [review]
LGTM.

**Pull request merged**